docs/getting_started.md

```diff
@@ -1,57 +1,337 @@
 
-Smartwatts is made for tracking the power consumption of processes on a
-machine.
-To install Smartwatts on a baremetal server or a PC run [the following
-script](script/smartwatts_install.sh) in a Terminal. Please notice that you will need [pip](https://pip.pypa.io/en/stable/installation/) or [docker](https://docs.docker.com/engine/install/) in order to use the Formula.
+### Example program to monitor
 
-The script explains what it will do and then pauses before it does it.
+[stress-ng](https://wiki.ubuntu.com/Kernel/Reference/stress-ng) can be used to 
+generate load on one's system.  
+An example usage, once installed :  
 
-Please notice that you need a **Linux distribution** in order to use the HWPC Sensor installed by the script as
-well as a **comptible Intel** (Sandy Bridge and newer) or **AMD Processor** (Zen). You also need [docker](https://docs.docker.com/engine/install/).  **Power/ARM/RISCV are not supported** architectures. HWPC Sensor will **not work on a Virtual Machine**. However, you can install the Formula by hand in a Virtual Machine if need it.
+```sh
+stress-ng --cpu 1 --timeout 5m
+```
 
+The PID can be found using :  
 
+```sh
+STRESS_PID="$(pgrep stress-ng)"
+```
 
-#### CGroups
-If you need to monitor a process or a group of process via SmartWatts by using HWPC Sensor **version 1.2 or older**, you can follow this [tutorial](reference/cgroup/cgroup.md). Please notice that **cgroup V1** is required **only** for HWPC Sensor **version 1.2 or older**. If you need to enable this `cgroup` version please follow this [tutorial](reference/cgroup/cgroup_v1_activation.md).    
+## Which components to get a complete stack  
 
-<!---
-## **Jouleit**
+If you wish to get started as soon as possible, the following docker-compose 
+file will allow you to deploy the following elements :  
 
-!!! note ""
-    for mesuring the energy consumption of a program
+1. A MongoDB instance to store the [Sensor](./reference/sensors/hwpc-sensor.md)
+Reports
 
-Jouleit is made for tracking the energy consumption of a program.
-Jouleit need `gawk` to run.
-You can get the script from the [github repository](https://github.com/powerapi-ng/jouleit)
-Start jouleit by using `./jouleit.sh cmd`.
--->
+2. An InfluxDB2 instance to store the [Formulas](./reference/formulas/smartwatts.md)
+
+3. An [HWPC-Sensor](./reference/sensors/hwpc-sensor.md) that outputs its 
+[HWPCReports](./reference/reports/report.md#HWPCReport) in a MongoDB Database, 
+within the HWPCReport Collection.  
+
+4. A [SmartWatts](./reference/formulas/smartwatts.md) that streams the 
+[HWPCReports](./reference/reports/report.md#HWPCReport) from the MongoDB 
+Database Collection, processes it and outputs its 
+[PowerReports](./reference/reports/report.md#PowerReports) in an InfluxDB2 
+Database within the PowerReport Collection  
+
+5. A Grafana instance to visualize the
+[PowerReports](./reference/reports/report.md#PowerReports) from the InfluxDB2 
+instance
+
```

Comment:
We have to say somewhere that  compatible OS + Kernel + Processor with sensor need to be used for a **local** deployment ?